NuGetResources/NuGetServiceInfo.cs

```diff
@@ -0,0 +1,103 @@
+﻿//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+namespace Microsoft.PackageManagement.NuGetProvider
+{
+    using Microsoft.PackageManagement.Provider.Utility;
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Parse and store a NuGet endpoint.
+    /// </summary>
+    public class NuGetServiceInfo
```

Comment:
can we consider it as internal?

Reply:
Must be public

---
In reply to: [154460179](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154460179) [](ancestors = 154460179)